From ca6a3916a3b4998ad15b9e1217fe032a0ddfd558 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Fri, 24 Jul 2020 20:55:37 +0200
Subject: [PATCH] test: do not rely on => being the prompt

In our tests we should use the customized prompt for testing.

Reported-by: Tom Rini <trini@konsulko.com>
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 test/py/tests/test_efi_loader.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/py/tests/test_efi_loader.py b/test/py/tests/test_efi_loader.py
index ca68626cec..fc8d6b8655 100644
--- a/test/py/tests/test_efi_loader.py
+++ b/test/py/tests/test_efi_loader.py
@@ -199,6 +199,6 @@ def test_efi_grub_net(u_boot_console):
     # Then exit cleanly
     u_boot_console.wait_for('grub>')
     u_boot_console.run_command('exit', wait_for_prompt=False, wait_for_echo=False)
-    u_boot_console.wait_for('=>')
+    u_boot_console.wait_for(u_boot_console.prompt)
     # And give us our U-Boot prompt back
     u_boot_console.run_command('')
-- 
2.27.0

